js/ui/components/clubhouse.js

```diff
@@ -38,7 +38,7 @@ const GtkNotificationDaemon = NotificationDaemon.GtkNotificationDaemon;
 
 const CLUBHOUSE_BANNER_ANIMATION_TIME = 0.2;
 
-var CLUBHOUSE_ID = 'com.hack_computer.Clubhouse';
+const CLUBHOUSE_ID = 'com.hack_computer.Clubhouse';
```

Comment:
I'd just remove this if you're replacing all the occurrences with a literal.

Reply:
Isn't that still used in https://github.com/endlessm/gnome-shell/blob/5f5ca0ca83116c0c88365fa59e6812228adae093/js/ui/components/clubhouse.js#L580 ?

Reply:
I added a new commit cleaning that up. We can squash this before merging but I wanted it to be a new commit to be looked at.